### ISSUE TITLE
Clickhouse keeper is broken in bitnami chart

### ISSUE THREAD

**Issue by mleklund** [stafftastic/jitsu-chart#19]
see:

https://github.com/bitnami/charts/issues/15935

Current fix is to switch to zookeper.  The issue has been languishing for quite a while, but just noted that they are increasing the priority.  To use zookeeper set your values to `clickhouse.keeper.enable = false` and `clickhouse.zookeeper.enabled = true`.

alternatively in values.yaml:
```
clickhouse:
  keeper:
    enabled: false
  zookeeper:
    enabled: true
```

**Comment by echozio** [comment]
Do you know if this made ClickHouse entirely unusable with CH Keeper? I'll have to treat the switch of the default values with a little more caution if it's possible the existing CH Keeper holds any data. As far as I'm aware there's no migration path from CH Keeper to Zookeeper.

**Comment by mleklund** [comment]
With the default config of shards=2, and replicas=3 I saw that databases never were created, so I went to create them manually since they were listed as not [created by prisma here](https://github.com/jitsucom/jitsu/tree/newjitsu/webapps/console/prisma), when attempting to create the tables manually I saw that the DDL operations would hang at 49%.  I spent way too much time digging through clickhouse logs and the error messages eventually led me back to [the mentioned bug report](https://github.com/bitnami/charts/issues/15935).   All that is to say, that I went about eliminating errors until I was able to have a working instance.

**Comment by mleklund** [comment]
I have been messing with the bitnami clickhouse chart, and it looks like the issue is that it create a set of keeper replicas for every shard.

**Comment by echozio** [comment]
Good job finding out all of this. I merged #20 to at least provide a working clickhouse deployment by default. At least there's a documented migration path from zookeeper to keeper for when keeper is fixed in Bitnami's chart. Thanks a lot for the contributions!
